build_tools/benchmarks/run_benchmarks.sh

```diff
@@ -26,8 +31,9 @@ TRACY_CAPTURE_TOOL="${IREE_TRACY_CAPTURE_TOOL}"
 E2E_TEST_ARTIFACTS_DIR="${1:-${IREE_E2E_TEST_ARTIFACTS_DIR}}"
 EXECUTION_BENCHMARK_CONFIG="${2:-${IREE_EXECUTION_BENCHMARK_CONFIG}}"
 TARGET_DEVICE_NAME="${3:-${IREE_TARGET_DEVICE_NAME}}"
-BENCHMARK_RESULTS="${4:-${IREE_BENCHMARK_RESULTS}}"
-BENCHMARK_TRACES="${5:-${IREE_BENCHMARK_TRACES}}"
+SHARD_INDEX="${4:-${IREE_SHARD_INDEX}}"
```

Comment:
I added `shard_index` as the argument right after `target_device_name` because that was the most logical to me. I'm not sure though if this will break someone. I can of course also add this as an argument at the end.

Reply:
I think it should be fine.